### DISCUSSION TITLE
en_US

### DISCUSSION THREAD

**Discussion by KN4MSH** [marcbat/svxlinkmanager#62] [Ideas]
Is it possible to change the main language from fr_FR to en_US, as an option ? Why it has to be limited to Franch language only ? It is svxlink manager, so it should be available for all EU users.

**Comment by marcbat** [comment]
73 KN4MSH,

thank you for your suggestion. There are plans to translate the software. But first, I just still have to finish fixing some bugs.
You can follow ticket # 37 which has been planned for version 3.2.0.

The behavior should be as follows.
The language displayed to the user will be that of the browser. The user can then select the language among the available languages and this language will become the default language for this user.

**Reply by KN4MSH** [reply]
The database file svxlinkmanager.db could be modified using DB.Browser.for.SQLite. The language field is the one to be changed from fr_FR to en_US. Install en_US/en_UK language package. New version 3.2.0 is lost in translation, it was abandoned in alpha stage.

**Comment by KN4MSH** [comment]
You made some progress already, Italian and English. Congratulations. Just waiting for a stable v3.2.0
